src/base/bittorrent/peerinfo.cpp

```diff
@@ -181,6 +181,31 @@ QString PeerInfo::client() const
     return QString::fromStdString(m_nativeInfo.client);
 }
 
+QString PeerInfo::peerId() const
```

Comment:
So this method doesn't actually return "peer id". It makes some UI related transformations.

Reply:
> You mean we probably want to rename it?

Probably, yes. But what exactly is it? "Decoded peer ID"?